### ISSUE TITLE
emacs-w32 with cygwin

### ISSUE THREAD

**Issue by omps** [jorgenschaefer/elpy#543]
Elpy Error

The backend encountered an unexpected error. This indicates a bug in
Elpy. Please open a bug report with the data below in the Elpy bug
tracker:

https://github.com/jorgenschaefer/elpy/issues/new

```
Error Message

[Error 3] The system cannot find the path specified: 'C:\\home\\username\\projects\\python'

Configuration

Virtualenv........: None
RPC Python........: 2.7.0 (/cygdrive/c/Users/username/localapps/Python27/pythonw)
Interactive Python: ipython (/usr/bin/ipython)
Emacs.............: 24.5.1
Elpy..............: 1.5.0 (Python), 1.7.1 (Emacs Lisp)
Jedi..............: 0.8.1-final0
Rope..............: 0.10.2
Importmagic.......: Not found
Syntax checker....: pyflakes (/usr/bin/pyflakes)

Traceback

Traceback (most recent call last):
  File "C:\Users\username\localapps\Python27\lib\site-packages\elpy\rpc.py", line 100, in handle_request
    result = method(*params)
  File "C:\Users\username\localapps\Python27\lib\site-packages\elpy\server.py", line 65, in rpc_init
    self.backend = ropebackend.RopeBackend(self.project_root)
  File "C:\Users\username\localapps\Python27\lib\site-packages\elpy\ropebackend.py", line 57, in __init__
    **prefs)
  File "C:\Users\username\localapps\Python27\lib\site-packages\rope\base\project.py", line 134, in __init__
    os.mkdir(self._address)
WindowsError: [Error 3] The system cannot find the path specified: 'C:\\home\\username\\projects\\python'
```

**Comment by jorgenschaefer** [comment]
Hello, and thank you for the report!

> `Elpy..............: 1.5.0 (Python), 1.7.1 (Emacs Lisp)`

I am not sure if this is related to the problem, but this is problematic. Could you make sure you use the same version of Elpy on both the Elisp and Python sides?

Other than that, this seems to be a duplicate of #517, so fixed in 7b75763, which should be in 1.8 when it comes out next month.

Are you encountering this problem a lot? Can you try and see if the development version fixes this problem, or would you rather wait until the next release?

**Comment by jorgenschaefer** [comment]
Lacking further updates, I assume the problem is solved. Closing this. Thanks for the report, and please do not hesitate to open a new one if you have further problems!
